tests/rt.conf

```diff
@@ -374,7 +364,7 @@ RUN | gnv1_c96_no_nest_debug                            | + hera hercules
 
 
 COMPILE | wam_debug | gnu | -DAPP=ATM -DCCPP_SUITES=FV3_GFS_v16_fv3wam -D32BIT=ON -DMULTI_GASES=ON -DDEBUG=ON | + hera hercules | fv3 |
-RUN | control_wam_debug                                 | + hercules                     | baseline |
+RUN | control_wam_debug                                 | + hera                     | baseline |
```

Comment:
So this test runs on hera now, not on hercules?

Reply:
I've changed this back to hercules. The test isn't running on hera right now.